contracts/ExampleNFT-v2-ContractInterface.cdc

```diff
@@ -0,0 +1,51 @@
+import NonFungibleToken from "./NonFungibleToken-v2.cdc"
+import MetadataViews from "./MetadataViews.cdc"
+
+pub contract interface NonFungibleTokenInterface {
+
+    /// Event that is emitted when a token is withdrawn,
+    /// indicating the owner of the collection that it was withdrawn from.
+    ///
+    /// If the collection is not in an account's storage, `from` will be `nil`.
+    ///
+    pub event Withdraw(id: UInt64, from: Address?, type: Type)
+
+    /// Event that emitted when a token is deposited to a collection.
+    ///
+    /// It indicates the owner of the collection that it was deposited to.
+    ///
+    pub event Deposit(id: UInt64, to: Address?, type: Type)
```

Comment:
Do you mean the storage path where the collection is stored? or the public receiver path? Those are the only paths that we can access from within the collection object, and I don't think either would be 100% accurate, because there are cases where they are stored in different places and don't have a way to pass that path information into the deposit function

Reply:
Gotcha.

Reply:
Can we include UUID as a field, since it is unique across all resources and we should access to it here?